src/autosuggest/autosuggest-option.tsx

```diff
@@ -13,7 +13,7 @@ import { HighlightType } from '../internal/components/options-list/utils/use-hig
 import { useInternalI18n } from '../i18n/context';
 
 export interface AutosuggestOptionProps extends BaseComponentProps {
-  nativeAttributes?: Record<string, any>;
+  nativeAttributes?: Record<string, string | boolean | number | undefined>;
```

Comment:
I couldn't find where we're actually providing the aria-label _value_, so potentially we could even remove it completely. I'd prefer to keep such a refactor separate though, so that e.g. we can revert it without reverting all the `any` fixes if needed.